### ISSUE TITLE
Only use pretrained model, Can I only download nuscenes test data?

### ISSUE THREAD

**Issue by yanshuaibupt** [mit-han-lab/bevfusion#61]

**Comment by kentang-mit** [comment]
I'm not sure whether it is possible to only download the nuScenes validation set. It seems to me that their API requires downloading the entire `v1.0-trainval`. If you only download `v1.0-mini`, I remember some people got it to work, but you will not be able to get the numbers presented in `README.md` because it's a different split.

**Comment by kentang-mit** [comment]
Temporarily closed due to inactivity.
